src/main/java/org/jenkinsci/test/acceptance/plugins/gitlab_plugin/GitLabOrganizationFolder.java

```diff
@@ -0,0 +1,44 @@
+package org.jenkinsci.test.acceptance.plugins.gitlab_plugin;
+
+import com.google.inject.Injector;
+import org.apache.commons.io.IOUtils;
+import org.jenkinsci.test.acceptance.po.*;
+
+import java.io.IOException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+
+@Describable("jenkins.branch.OrganizationFolder")
+public class GitLabOrganizationFolder extends Folder {
+    public GitLabOrganizationFolder(Injector injector, URL url, String name) {
+        super(injector, url, name);
+    }
+
+    public void create(String owner) {
+        control(by.path("/hetero-list-add[navigators]")).click();
+        find(by.partialLinkText("GitLab Group")).click();
+        find(by.path("/navigators/projectOwner")).sendKeys(owner);
+    }
+
+    @Override
+    public URL getConfigUrl() {
+        return null;
+    }
+
+    public String getCheckLog() {
+        try {
+            return IOUtils.toString(url("computation/console").openStream(), StandardCharsets.UTF_8);
```

Comment:
this should ideally use webdriver so that you can see what is happening when it happens.  Additionally, if you have enabled security this would fail as it is not passing any authentication which would be in the session for the WebDriver